README.md

```diff
@@ -1047,6 +1047,10 @@ sensu_bonsai_asset { 'sensu/sensu-pagerduty-handler':
 
 The following Bolt tasks are provided by this Module:
 
+**sensu::backend\_upgrade**: Perform backend upgrade via `sensu-backend upgrade` command.
+
+Example: `bolt task run sensu::backend_upgrade --targets sensu_backend`
```

Comment:
I believe you only run on *one* of the backends, so the target will be different.

Reply:
Right, maybe I should change `--targets sensu_backend` to `--targets sensu-backend.example.com` to make it clear it's FQDN but IIRC in Bolt a target name isn't necessarily a FQDN, it could be something else like Sensu entity name if using this modules target resolver.  But yes, I believe you only run the migration on one of the backends if using a cluster setup, as the change will replicate to other backends.